lsp-haskell.el

```diff
@@ -228,6 +228,25 @@ if projectile way fails"
           (user-error "Couldn't find cabal file, using: %s" dir)
         dir))))
 
+
+;; ---------------------------------------------------------------------
+;; Interactive commands
+
+(defun lsp-haskell-case-split ()
+  "Case split on an identifier.
+To use this, place make sure the point is over a type hole."
+  (interactive)
+  (let* ((case-split-regex "Case split on \\(.*\\)")
```

Comment:
This feels a little fragile. I *think* the right way to do this is to use code action kinds: they're explicitly there to allow filtering by the client (i.e. us). We might need to teach the tactics plugin to add a kind, though.

Reply:
I can update the tactics plugin with kinds, good call!